server/db/services/calls.js

```diff
@@ -53,5 +55,10 @@ export default {
       .save({ status: 'AVAILABLE', user_id: null }, { patch: true })
       .then(call => call)
       .catch(err => console.log('error in calls service when releasing call: ', err));
+  },
+
+  hangUpCall: (params) => {
+    const { call_sid } = params;
+    return hangUp(call_sid);
```

Comment:
Is this the pattern we're supposed to be using? I've been importing twilio utility functions at the controller.